### PR TITLE
Subject: fix_5863 followers_list unassigned variable

### LINKED ISSUE #5821: Grabbed 0 usernames from `Followers` in 0.01 seconds
## Expected Behavior
Selenium will open Firefox, Log in to the account, Get the list of the followers of the user required.

## Current Behavior
Selenium will open Firefox, will open the window of the profile I am checking, won't log in into my account, and will open:
`view-source:https://www.instagram.com/graphql/query/?query_hash=37479f2b8209594dde7facb0d904896a&variables={%22id%22:%20%228638029069%22,%20%22first%22:%2050}`
In which I get the following data:
`{"data":{"user":{"edge_followed_by":{"count":360,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},"status":"ok"}`
and then the output in the program is: `Grabbed 0 usernames from `Followers` in 0.01 seconds`
and then program will just exit.

### LINKED ISSUE #5863: interact_user_followers generates endless GraphQL queries
## Expected Behavior
Collect amount of followers of a specific user
## Current Behavior
Last Log: **GraphQL query hash: [...]**
Generates endless GraphQL requests within the browser like:
`{"data":{"user":{"edge_followed_by":{"count":1490809,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]},"edge_mutual_followed_by":{"count":0,"edges":[]}}},"status":"ok"}`
with the URL:
`view-source:https://www.instagram.com/graphql/query/?query_hash...`
Nothing more happens. No error or termination. Tried it many times, a few hours, even on different machines
## InstaPy configuration
Python3, InstaPy Version: 0.6.12
(dont_likes & targets are lists specified before)

```
with smart_run(session):
    session.set_dont_like(dont_likes)
    session.set_simulation(enabled=True)
    session.set_relationship_bounds(enabled=True,
                                    potency_ratio=None,
                                    delimit_by_numbers=True,
                                    max_followers=None,
                                    max_following=None,
                                    min_followers=200,
                                    min_following=None,
                                    min_posts=5)

    session.set_skip_users(skip_private=True,
                           skip_no_profile_pic=True,
                           skip_business=False)

    session.set_user_interact(amount=3, randomize=True, percentage=80,
                              media='Photo')
    session.set_do_like(enabled=True, percentage=90)
    session.set_do_comment(enabled=True, percentage=15)
    session.set_comments(comments, media='Photo')```

    session.interact_user_followers(targets,
                                  amount=random.randint(30, 60),
                                  randomize=True)

### PR DESCRIPTION
**Pull request by elulcao**
Problem: followers_list cleared due to double declaration.
Solution: followers_list is now defined once in `get_users_through_dialog_with_graphql`

ie:
`1605144344786 is 19:25:44.786 GMT-06:00`
```
1605144344786	Marionette	DEBUG	0 <- [1,649,null,{"value":"{\"data\":{\"user\":{\"edge_followed_by\":{\"count\":364124,\"page_info\":{\"has_next_page\":true,\"en ... kS&_nc_tp=25&oh=e5318dac5748263890cb65&oe=5FD5B830\",\"username\":\"dirscejeff\"}}}}]}}},\"status\":\"ok\"}"}]
1605144344787	webdriver::server	DEBUG	<- 200 OK {"value":"{\"data\":{\"user\":{\"edge_followed_by\":{\"count\":364124,\"page_info\":{\"has_next_page\":true,\"end_cursor\":\"QVFEeVM5bWE0dFJ5Nmlqa0taYmFVQU05N21LQVpDRVE1NVkzbDVpTG5hN0R3N1pKNWNPd24TcmNYeg==\"},\"edges\":[{\"node\":{\"id\":\"1999851\",\"username\":
```
```
INFO [2020-11-11 19:25:34] [internet]  User 'internet' [2/6]
INFO [2020-11-11 19:25:41] [internet]  GraphQL query hash: [c76146de903be841dd25a]
INFO [2020-11-11 19:25:44] [internet]  To be followed: [1/5/internet]
INFO [2020-11-11 19:25:44] [internet]  To be followed: [2/5/internet]
INFO [2020-11-11 19:25:44] [internet]  To be followed: [3/5/internet]
INFO [2020-11-11 19:25:44] [internet]  To be followed: [4/5/internet]
INFO [2020-11-11 19:25:44] [internet]  To be followed: [5/5/internet]
INFO [2020-11-11 19:25:44] [internet]  Grabbed 5 usernames from 'internet `Followers` to do following
```

### CODE REVIEW

**Comment by utsav1810** [comment]
I tried your changes but it is still going in an endless loop of GraphQL query hash hits on the browser.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,20 @@ _The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
 
 ## [0.6.13] - UNRELEASED
+
 ### Added
+
 - Handle situation if stale elements detected when cookie cannot be loaded
 
+### Fixed
+
+- Fixed typo in message when a video is tried to like.
```
Comment (breuerfelix) [inline CHANGELOG.md:16]: i just can't find where you fix this issue in this PR :D

**Review by breuerfelix — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (timgrossmann) [reply CHANGELOG.md:16]: Hahaha, had the same issue, he added it from another pull request 😄

**Review by timgrossmann — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (breuerfelix) [reply CHANGELOG.md:16]: good to know :D

**Comment by elulcao** [comment]
Correct,

The other pull request was merge via “Merge pull request #5869 from JackBonnellDevelopment/master”,
to avoid doble changes in CHANGELOG I placed both comments in this PR.
Next time I will share in detail if this situation occurs.

**Comment by breuerfelix** [comment]
haha no worries mate! that is totally fine :) better insert it than ignoring it!

**Pull request merged**